src/io/flutter/view/FlutterViewFactory.java

```diff
@@ -32,9 +31,6 @@ private static void initFlutterView(@NotNull Project project, FlutterViewMessage
 
   @Override
   public void createToolWindowContent(@NotNull Project project, @NotNull ToolWindow toolWindow) {
-    //noinspection CodeBlock2Expr
-    DumbService.getInstance(project).runWhenSmart(() -> {
```

Comment:
Wonder if there are other places we can make similar simplifications.  I don't think being "smart" gets us much.